src/wormhole/session/tile/plant/mod.rs

```diff
@@ -10,6 +10,9 @@ use summon::summon;
 mod rub;
 use rub::rub;
 
+mod nickname;
+use nickname::nickname;
```

Comment:
the other one is named rename_gotchi, but this one is named "nickname". I know this is a bit of a nitpick, but could we be consistent here for maintenance's sake?